tests/ui/associated-type-bounds/return-type-notation/bad-inputs-and-output.rs

```diff
@@ -0,0 +1,17 @@
+// edition: 2021
+
+#![feature(return_type_notation, async_fn_in_trait)]
+//~^ WARN the feature `return_type_notation` is incomplete
+//~| WARN the feature `async_fn_in_trait` is incomplete
+
+trait Trait {
+    async fn method() {}
+}
+
+fn foo<T: Trait<method(i32): Send>>() {}
+//~^ ERROR argument types not allowed with return type notation
```

Comment:
If I get this right, even if `async fn method(_: i32)` the bound is `method(): Send`?

Reply:
Correct

Reply:
nevermind, latest commit has made it `method(..): Send` now